### DISCUSSION TITLE
How to increase query log size?

### DISCUSSION THREAD

**Discussion by pcparklin** [azerothcore/azerothcore-wotlk#16878] [General]
Hello.

I tried to increase the size of query log,
and modified the variable MAX_QUEST_LOG_SIZE in QuestDef.h
`#define MAX_QUEST_LOG_SIZE 26`
But the limit shown in quest log menu is still 25,
What else should I do?

**Comment by Winfidonarleyan** [comment]
This is a limitation on the part of the client, nothing can be done

**Reply by welder1976** [reply]
need rework interface
